services/motion/motion.go

```diff
@@ -67,6 +66,20 @@ type Service interface {
 	) (*referenceframe.PoseInFrame, error)
 }
 
+// MotionConfiguration specifies how to configure a call
+//
+//nolint:revive
+type MotionConfiguration struct {
+	VisionSvc           []resource.Name
+	PositionPollingFreq float64
+	ObstaclePollingFreq float64
+	PlanDeviationMeters float64
+	ReplanCostFactor    float64
+	LinearMetersPerSec  float64
+	AngularMetersPerSec float64
+	Extra               interface{}
```

Comment:
This is a little awkward to have here. Since each motion call also has extras; this now means that each motion call will have *two* different sets of extras, which would need to be rectified, and could be contradictory. Would prefer if we go back and ret-con this field from the scope doc.

Also, extras are of type `map[string]interface{}`